src/storage/mvcc/txn.rs

```diff
@@ -215,8 +216,10 @@ impl<S: Snapshot> MvccTxn<S> {
                         // None: related Rollback has been collapsed.
                         // Rollback: rollback by concurrent transaction.
                         info!(
-                            "txn conflict (lock not found), key:{}, start_ts:{}, commit_ts:{}",
-                            key, self.start_ts, commit_ts
+                            "txn conflict (lock not found)";
+                            "key" => hex::encode_upper(key.as_encoded()),
```

Comment:
Can we use escape instead of hex?

Reply:
We need to follow the "Unified Key Format" RFC to use the hex format.

Reply:
@zhangjinpeng1987 

Please remember to change other places which use escape.